tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "`% cp /home/shared/data/submit_em.sh .`\n",
+    "\n",
+    "You can explore its contents and see that the last line in the file contains the `mdrun` command by typing: \n",
+    "\n",
+    "`% cat submit_em.sh`\n",
+    "\n",
+    "The file contents will be printed on your terminal.\\\n",
+    "Now submit it to the cluster queue.\n",
+    "\n",
+    "`% sbatch submit_em.sh`\n",
+    "\n",
+    "It will take a few minutes to run, depending on the waiting times of the queue. Check on the status of the run by typing:\n",
+    "\n",
+    "`% squeue -u username`\n",
+    "\n",
+    "Remember to replace `username` with your own username! It should print something like this:\n",
+    "\n",
+    "`             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON) \n",
+    "           1311337       htc       EM bioc1550 PD       0:00      1 (Priority) \n",
+    "`\n",
+    "\n",
+    "The `PD` in the fifth column denotes that the job is in the queue and has not started running yet. It will change to `R` once it starts running:\n",
+    "\n",
+    "`             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON) \n",
+    "           1311337       htc       EM bioc1550  R       1:26      1 arcus-htc-node110 \n",
+    "`\n",
+    "\n",
+    "If the job has finished (or if it has failed), the above command will print nothing:\n",
+    "\n",
+    "`JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)`\n",
+    "\n",
+    "If you made a mistake and need to cancel this job, type:\n",
+    "\n",
+    "`% scancel JOBID`\n",
+    "\n",
+    "`JOBID` should be replaced by the ID of this particular job (printed in the first column; see above).\n",
+    "\n",
+    "While the job is running, you can monitor its progress by typing: \n",
+    "\n",
+    "`% tail -n 12 em.log`\n",
+    "\n",
+    "This will show you what step the simulation is at or if it has finished.\\\n",
+    "Once the energy minimization has finished, we can examine it in terms of the potential energy versus the number of minimization steps:\n",
+    "\n",
+    "`% gmx energy -s em.tpr -f em.edr -o em_potential_energy.xvg`\n",
+    "\n",
+    "type in 10 when prompted (should correspond to potential energy from the list of options presented), then a zero to finish.\\\n",
+    "The plot should look like this one:\n",
+    "\n",
+    "<img src=\"em_potential_energy.png\" width=500 height=400/>\n",
+    "\n",
+    "### Production Run\n",
+    "\n",
+    "At this stage, we would normally run a short simulation where the protein atoms are restrained while the water molecules\n",
```

Comment:
Can we point them to the md tutorials website for a more detailed step by step process where they do this?

Also do we need to explain why we would do this normally?

Reply:
Briefly explained. Will be discussed during plecture too